box-share-sdk/src/main/java/com/box/androidsdk/share/usx/adapters/CollaboratorsAdapter.java

```diff
@@ -17,30 +19,37 @@
 import com.box.androidsdk.share.CollaborationUtils;
 import com.box.androidsdk.share.R;
 import com.box.androidsdk.share.api.ShareController;
+import com.box.androidsdk.share.databinding.UsxListItemCollaborationBinding;
+import com.box.androidsdk.share.vm.BaseShareVM;
 import com.eclipsesource.json.JsonObject;
 
+import java.io.Serializable;
 import java.util.ArrayList;
 
 public class CollaboratorsAdapter extends BaseAdapter {
 
     private ArrayList<BoxCollaboration> mItems = new ArrayList<BoxCollaboration>();
     private Context mContext;
-    private BoxCollaborationItem mFolder;
-    private ShareController mController;
     private final BoxCollaborator mAnotherPersonCollaborator;
 
+    String userId;
+
+    private BaseShareVM mBaseShareVM;
 
-    public CollaboratorsAdapter(Context context, BoxCollaborationItem collaborationItem, ShareController controller) {
+    public CollaboratorsAdapter(Context context, BaseShareVM baseShareVM) {
         super();
         mContext = context;
-        mFolder = collaborationItem;
-        mController = controller;
+        mBaseShareVM = baseShareVM;
         // This item is used for displaying users that do not have a box account have been invited as a collaborator
         JsonObject jsonObject = new JsonObject();
         jsonObject.add(BoxCollaborator.FIELD_NAME, mContext.getString(R.string.box_sharesdk_another_person));
         mAnotherPersonCollaborator = new BoxUser(jsonObject);
     }
 
+    public BoxItem getItem() {
```

Comment:
Rename this to `getShareItem()`. Since this class is called CollaboratorsAdapter which will contain a list of collaborator items, it is easy to believe that a getItem() call will return a `BoxCollaboration` object and not a BoxItem.